### DISCUSSION TITLE
支付宝购买lantern一年版本，付费后未升级为专业版

### DISCUSSION THREAD

**Discussion by lxhlxh8999** [getlantern/lantern-forum-cn#884] [Q&A]
支付宝购买lantern一年版本，付费后未升级为专业版
@Mike777Khoo
![a73789482a8e6e617ef3b5d14e2aecc](https://github.com/user-attachments/assets/6039d1c8-a19a-4869-88bb-dae0bf9c6200)

**Comment by MaxMKMax** [comment]
@lxhlxh8999 好的. 请您也提供我账号邮箱. 我帮您安排一下.

**Reply by lxhlxh8999** [reply]
lxhxh87@sina.com，谢谢

**Reply by shenmz233** [reply]
我也一样，帮忙处理一下谢谢， shenmz233@163.com

**Reply by MaxMKMax** [reply]
@shenmz233 您好. 请您重新开个帖子在提供我支付单据, 这样会比较好. 谢谢!

**Comment by lxhlxh8999** [comment]
你好，问题尚未解决，是会发给我激活码吗？
----- 原始邮件 -----
发件人：Mike K ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：lxhlxh8999 ***@***.***>,  Mention ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_支付宝购买lantern一年版本，付费后未升级为专业版_(Discussion_#884)
日期：2024年09月24日 23点19分


@lxhlxh8999 好的. 请您也提供我账号邮箱. 我帮您安排一下.

**Reply by MaxMKMax** [reply]
@lxhlxh8999 我已经帮您提交问题了. 请您稍等 2 天. 谢谢您的耐心等待!

**Comment by lxhlxh8999** [comment]
有回复吗？问题尚未解决
----- 原始邮件 -----
发件人：Mike K ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：lxhlxh8999 ***@***.***>,  Mention ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_支付宝购买lantern一年版本，付费后未升级为专业版_(Discussion_#884)
日期：2024年10月01日 04点28分


@lxhlxh8999 我已经帮您提交问题了. 请您稍等 2 天. 谢谢您的耐心等待!

**Comment by MaxMKMax** [comment]
@lxhlxh8999 您好. 这是您的激活码:
J3CYD-6TFMY-29V8W-H4QHY-2MRBD
